docker/buildmaster/Dockerfile

```
@@ -1,3 +1,29 @@
+FROM alpinelinux/build-base:latest-x86_64 AS apk-builder
+
+RUN abuild-apk add -u \
+	gcc \
+	git \
+	linux-headers \
+	lua5.3-dev \
+	lua5.3-lzlib \
+	make \
+	musl-dev \
+	openssl-dev \
+	openssl-libs-static \
+	zlib-dev \
+	zlib-static \
+	zstd-dev \
+	zstd-static
+
+ARG APK_TOOLS_COMMIT=54caa31be633efc5f655700b77af290124f71689
```

I guess you may also use the latest APK version, I'll bump the one in openwrt.git soonish too...
```suggestion
ARG APK_TOOLS_COMMIT=74de0e9bd73d1af8720df40aa68d472943909804
```